examples/twittermap/web/public/javascripts/timeseries/controllers.js

```diff
@@ -80,6 +84,16 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
       }
     };
 
+    function eventHandler(){
+      //reset time series state
+      $scope.timeseriesState = 1;

```

Comment:
Why the reset value is 1 but the initial value is 0?

Reply:
When `resultArray` is created, watch function will be triggered, even if we didn't actually receive a new result. So I want to distinguish the initializing state and the first-time receive query result state.